docs/partners/partner-config.md

```diff
@@ -0,0 +1,197 @@
+
+:::
+
+## Introduction
+
+The Partner Configuration allows Partners to make themselves known to the Camino Network as service providers, or as looking for services from other Partners in the Network. Upon matching between services wanted and offered by two parties, such services will be then traded on Camino Messenger.
+
+Services are defined within the concept of Message Types: you can find [here](../camino-messenger/introduction#main-message-types) a complete documentation with pointers to the list of available types. As a distributor or a supplier, a Partner usually wants _and_ offers several services at the same time.
+For example:
+
+A hospitality distributor, e.g. a tour operator, might want to consume:
+
+- AccommodationProductInfoService
+- AccommodationProductListService
+- AccommodationSearchService
+- ValidationService
+- MintService
+
+A hospitality supplier, e.g. a hotel chain, might offer the same services for consumption.
+
+A the same time, the tour operator might want to consume from an info provider:
+
+- DestinationInformationService
+
+Which also requires that the consumer exposes a callback method to actually receive events updates:
+
+- DestinationInformationCallbackService (COMING SOON)
+
+And the information supplier has then to conversely offer / consume the respective services. It can become a bit intricated for partners wanting and offering multiple services: the Camino Foundation and the community are available to support with best practices for your specific case and addressing implementation on our [Discord](https://discord.gg/camino).
+
+## Actors and components
+
+Before dwelwing into the details of how to configure a Partner, it is useful to go through an explanation of involved actors (human and software components, onchain and offchain) and their specific responsibilities in the Partner's operation on Camino Messenger.
+
+```mermaid
+graph TD
+    CFO([<i>CFO</i>]) --> |owns| FinanceWallet
+    CTO([<i>Tech power user</i>]) --> |owns| TechWallet
+    CTO --> |deploys| Bot1
+    CTO --> |deploys| Bot2
+    CTO --> |owns| Bot1Wallet
+    CTO --> |owns| Bot2Wallet
+    FinanceWallet[<b>Finance wallet</b><br/>- Holds main org funds, allocates them to trading on Messenger] <-.-> CMAccount
+    TechWallet[<b>Tech wallet</b><br/>- Creates and manages the CMA, paying gas fees<br/>- Can to withdraw money from CMA] --> CMAccount
+    CMAccount[<b>CMA smartcontract</b><br/>- Holds trading funds<br/>- Specifies services and fees<br/>- Specifies accepted currencies<br>- Registers bots] --> Bot1Wallet
+    CMAccount --> Bot2Wallet
+    Bot1(<i>Bot1</i>) --> |uses| Bot1Wallet[<b>Bot1 wallet</b><br/>- Can withdraw limited amounts from CMA<br/>- As <i>supplier</i>: cashes in service fee checks, pays gas fees for minting booking tokens<br/>- As <i>distributor</i>: pays for service fees and buying booking tokens]
```

Comment:
> As distributor: pays for service fees and buying booking tokens

This and also the explanations below about the bots are a little off. Bot's pay only for the gas costs for the cheque cash-in and BookingToken buying operations. The actual amount is paid from the CM Account.

Reply:
ok, correcting.